servicetest/dbus/test_dbus.py

```diff
@@ -98,7 +98,6 @@ def test_query_in(self):
             finally:
                 ca.terminate()
 
-    @pytest.mark.skip(reason="See reported issue about this")
```

Comment:
In that case I think we need somewhere better to reference, or add a better comment about what's actually wrong. Anyways, it was removed so no point debating it I guess ;)